test/edu/psu/sweng500/team8/puzzleGenerator/SolutionGeneratorTests.java

```diff
@@ -234,6 +234,7 @@ private static Board boardGeneration() {
 		boolean keepGoing = true;
 		int breakCounter = 0;
 
+		//FIXME: Why not use a for loop with a break statement instead of a separate counter and a flag??

```

Comment:
Scott, any comment on this? This seems like a convoluted way to do a simple loop